### PR TITLE
fix: blacklist usdc wormholes

### PR DESCRIPTION
**Pull request by 0xdef1cafe**
these assets being blacklisted are USDC wormhole contracts. you can't hold them as an asset and we don't support the
wormhole(s). they're polluting the asset list and are confusing to users.

### CODE REVIEW

**Comment by 0xdef1cafe** [comment]
https://etherscan.io/address/0x566957ef80f9fd5526cd2bef8be67035c0b81130

https://etherscan.io/address/0x7cd167b101d2808cfd2c45d17b2e7ea9f46b74b6

File: packages/asset-service/src/generateAssetData/blacklist.json [hunk 1]
```diff
@@ -8,5 +8,7 @@
   "eip155:1/erc20:0x1412eca9dc7daef60451e3155bb8dbf9da349933",
   "eip155:1/erc20:0x34278f6f40079eae344cbac61a764bcf85afc949",
   "eip155:1/erc20:0xc12d1c73ee7dc3615ba4e37e4abfdbddfa38907e",
-  "eip155:1/erc20:0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"
+  "eip155:1/erc20:0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee",
+  "eip155:1/erc20:0x566957ef80f9fd5526cd2bef8be67035c0b81130",
```
Comment (0xdef1cafe) [inline packages/asset-service/src/generateAssetData/blacklist.json:12]: https://etherscan.io/address/0x566957ef80f9fd5526cd2bef8be67035c0b81130

File: packages/asset-service/src/generateAssetData/blacklist.json [hunk 2]
```diff
@@ -8,5 +8,7 @@
...
+  "eip155:1/erc20:0x7cd167b101d2808cfd2c45d17b2e7ea9f46b74b6"
```
Comment (0xdef1cafe) [inline packages/asset-service/src/generateAssetData/blacklist.json:13]: https://etherscan.io/address/0x7cd167b101d2808cfd2c45d17b2e7ea9f46b74b6

**Comment by kaladinlight** [comment]
Shouldn't we just filter out these wormhole assets from the coingecko asset service? Going to be a lot of blacklisting this way and applicable to both eth and avax

**Comment by 0xdef1cafe** [comment]
> Shouldn't we just filter out these wormhole assets from the coingecko asset service? Going to be a lot of blacklisting this way and applicable to both eth and avax

there's insufficient info - both name and symbol are just `USDC`, otherwise yeah 

also - that means we have two places with blacklist logic

**Comment by kaladinlight** [comment]
> > Shouldn't we just filter out these wormhole assets from the coingecko asset service? Going to be a lot of blacklisting this way and applicable to both eth and avax
> 
> there's insufficient info - both name and symbol are just `USDC`, otherwise yeah
> 
> also - that means we have two places with blacklist logic

At the time of interacting with coingecko api directly, it seems like their `id` always contains `-wormhole-`

```
    {
      "id": "usd-coin-wormhole-from-bsc",
      "symbol": "usdcbs",
      "name": "USD Coin (Wormhole from BSC)",
      "platforms": {
        "solana": "FCqfQSujuPxy6V42UvafBhsysWtEq1vhjfMN1PUbgaxA",
        "oasis": "0x4ca2a3de42eabc8fd8b0ac46127e64db08b9150e",
        "avalanche": "0x6145e8a910ae937913426bf32de2b26039728acf",
        "terra": "terra1yljlrxvkar0c6ujpvf8g57m5rpcwl7r032zyvu",
        "ethereum": "0x7cd167b101d2808cfd2c45d17b2e7ea9f46b74b6"
      }
    },
```

**Review by kaladinlight — APPROVED** [review]

**Review by 0xApotheosis — APPROVED** [review]

**Pull request merged**